internal/service/event_post_processing.go

```diff
@@ -1113,6 +1083,20 @@ func (s *eventPostProcessingService) enrichAnalyticsWithFeatureAndMeterData(ctx
 				item.MeterID = meter.ID
 				item.EventName = meter.EventName
 				item.AggregationType = meter.Aggregation.Type
+
+				// Calculate cost if price is available for this meter
```

Comment:
everywhere cost calculation needs to be price / subscription line item start end duration